packages/yoastseo/src/scoring/assessments/inclusiveLanguage/configuration/disabilityRulesData.js

```diff
@@ -0,0 +1,71 @@
+import passiveAuxiliaries from "../../../../languageProcessing/languages/en/config/internal/passiveVoiceAuxiliaries";
+import { flatMap } from "lodash-es";
+
+const intensifiersAndAdverbs = [ "so", "very", "a bit", "really", "pretty", "kind of", "that", "too", "totally", "completely", "absolutely", "even",
+	"also", "as" ];
+const formsOfToDrive = [ "driving", "drive", "drove", "drives", "driven" ];
+const objectPronouns = [ "me", "you", "them", "him", "her", "someone", "somebody", "anyone", "anybody", "everyone", "everybody" ];
+// Remove 'having' and 'what's' from the auxiliaries. We don't want to use them for any rules.
+const formsOfToBeAndToGet = passiveAuxiliaries.slice( 0, -2 );
+// The negated forms are: "isn't", "weren't", "wasn't", "aren't".
+const negatedFormsOfToBe = formsOfToBeAndToGet.slice( 19, 23 );
+const nonNegatedFormsOfToBeToGet = formsOfToBeAndToGet.slice( 0, 19 ).concat( formsOfToBeAndToGet.slice( 23, 25 ) );
+
+/**
+ * Creates an array of all possible combinations of strings from two arrays.
+ * For example, if array 1 is [ "fluffy", "cute" ] and array 2 is [ "cats", "dogs" ],
+ * it creates the array [ "cute cats", "cute dogs", "fluffy cats", "fluffy dogs" ].
+ *
+ * @param {string[]} arrayOne The first array of strings.
+ * @param {string[]} arrayTwo The second array of strings.
+ *
+ * @returns {string[]} An array containing all possible combinations of strings from two arrays.
+ */
+const createCombinationsFromTwoArrays = function( arrayOne, arrayTwo ) {
+	return flatMap( arrayOne, stringFromArrayOne => flatMap( arrayTwo, stringFromArrayTwo => `${stringFromArrayOne} ${stringFromArrayTwo}` ) );
+};
+
+/*
+ * Create an array of strings that should precede 'OCD' and the non-negated version of 'crazy about'.
+ * It includes all forms of 'to be/to get', optionally followed by an intensifier or a specific adverb (e.g., 'is', 'is very', 'are even').
+ */
+const combinationsOfToBeAndIntensifier = createCombinationsFromTwoArrays( formsOfToBeAndToGet, intensifiersAndAdverbs );
```

Comment:
In this case, we're using the `formsOfToBeAndToGet` list, which actually includes "isn't", "weren't", "wasn't", and "aren't". If I understand this correctly, this means that in `to be crazy about` identifier, we actually also target some forms of `to be not crazy about` ?

Reply:
Wow, that's also a good catch! 🌟